js/create-thumbnails.js

```diff
@@ -1,22 +1,22 @@
-import { createDescriptions } from './data.js';
+//import { createDescriptions } from './data.js';
 
 const allPictures = document.querySelector('.pictures');
 const usersPicturesTemplate = document.querySelector('#picture').content.querySelector('.picture');
 
-const usersPictures = createDescriptions();
+const renderPicture = (usersPictures) => {
```

Comment:
```suggestion
const renderPictures = (pictures) => {
```
всё таки много картинок рисуешь, а не одну